src/ir_Daikin.h

```diff
@@ -344,8 +357,7 @@ const uint8_t kDaikin176ByteSwingH = 18;
 const uint8_t kDaikin176SwingHAuto =  0x5;
 const uint8_t kDaikin176SwingHOff = 0x6;
 
-// Another variant of the protocol for the Daikin BRC52B63 remote.
-// Ref: https://github.com/crankyoldgit/IRremoteESP8266/issues/827
+
```

Comment:
... but extra line here ...

Reply:
I'll go with double blank lines between "protocols", to be more consistent.

Reply:
sounds good